andela/learning-map-front
# #149886251 Contribute Instructions Card

#### What does this PR do?
Allows users see instructions on how to make contributions to the learning map.

#### Description of Task to be completed?
- Create a card that shows how to make contributions

#### How should this be manually tested?
After cloning the repo

- Run yarn install
- Run yarn start
- Access the landing page via `localhost:3000` and click on the `LOGIN TO CONTRIBUTE` button.
- Navigate to contributions page to see card

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#149886251](https://www.pivotaltracker.com/story/show/149886251)

#### Screenshot
<img width="1138" alt="screen shot 2017-08-03 at 21 47 03" src="https://user-images.githubusercontent.com/24937777/28943050-5698efde-7895-11e7-97cc-86c729db2138.png">


#### Question
N/A
